cmd/kuberhealthy/config.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"io/ioutil"
+
+	"github.com/fsnotify/fsnotify"
+	log "github.com/sirupsen/logrus"
+	"sigs.k8s.io/yaml"
+)
+
+// Config holds all configurable options
+type Config struct {
+	kubeConfigFile            string
+	ListenAddress             string `yaml:"listenAddress"`
+	EnableForceMaster         bool   `yaml:"enableForceMaster"`
+	LogLevel                  string `yaml:"logLevel"`
+	InfluxUsername            string `yaml:"influxUsername"`
+	InfluxPassword            string `yaml:"influxPassword"`
+	InfluxURL                 string `yaml:"influxURL"`
+	InfluxDB                  string `yaml:"influxDB"`
+	EnableInflux              bool   `yaml:"enableInflux"`
+	ExternalCheckReportingURL string `yaml:"externalCheckReportingURL"`
+}
+
+// Load loads file from disk
+func (c *Config) Load(file string) error {
+
+	b, err := ioutil.ReadFile(file)
+	if err != nil {
+		return err
+	}
+
+	return yaml.Unmarshal(b, c)
+}
+
+// changeConfig creates a watcher and can be used to notify of change to configmap
+func changeConfig() {
```

Comment:
```suggestion
func configChangeNotifier(configPath string) (chan struct{}) {
```

We want this `func` to return a channel that can be listed to in order to know when the file has changed.  The upstream code will monitor this channel and when a signal comes through, initiate reloading the main `Kuberhealthy` struct.

Reply:
Changes made with 6bf8b54